### PR TITLE
Etun 203

### PR DESCRIPTION
**Pull request by LunaSquee**

### CODE REVIEW

File: angular-fe/src/app/modules/certificates/certificates.utility.ts [hunk 1]
```diff
@@ -151,4 +153,52 @@ export class CertificatesUtility {
     };
   }
 
+  /**
+   * Return string list of educational institutions from a final document
+   * @param document Certificate Document
+   * @returns String list of institutions, separated by comma
+   */
```
Comment (madispold) [inline angular-fe/src/app/modules/certificates/certificates.utility.ts:160]: Hella nice!

File: angular-fe/src/app/modules/certificates/components/certificate-detailed/certificate-detailed.component.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import { CertificateDocument } from '../../models/interfaces/certificate-document';
+
+@Component({
+  selector: 'certificate-detailed',
+  templateUrl: './certificate-detailed.component.html',
+  styleUrls: ['./certificate-detailed.component.scss'],
+})
+export class CertificateDetailedComponent {
+
+  @Input() documents: any = {};
```
Comment (madispold) [inline angular-fe/src/app/modules/certificates/components/certificate-detailed/certificate-detailed.component.ts:18]: No type/interface for this?

File: angular-fe/src/app/modules/certificates/certificates.utility.ts [hunk 3]
```diff
@@ -151,4 +153,52 @@ export class CertificatesUtility {
...
+  public static parseSupplementaryDocuments(
+    initial: FormattedCertificateDocumentData,
+    response: CertificateDocumentResponse[],
+    data: CertificateData,
+  ): FormattedCertificateDocumentData {
+    for (const doc of response) {
+      if (doc.document.type.indexOf('SUPPLEMENT') !== -1) {
+        initial.supplement = doc.document;
+        initial.supplement.content = JSON.parse(doc.document.content as string);
```
Comment (madispold) [inline angular-fe/src/app/modules/certificates/certificates.utility.ts:191]: Might consider adding as a separate function for better readabilty... but really a preference.

File: angular-fe/src/app/modules/certificates/components/certificate-detailed/certificate-detailed.component.ts (hunk 2 above)
Comment (LunaSquee) [reply angular-fe/src/app/modules/certificates/components/certificate-detailed/certificate-detailed.component.ts:18]: Adding a type here would require a lot of casting thus is not reasonable. Type can be assumed from the input anyway if needed.

**Pull request merged**